### PR TITLE
James 2558 backport

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

**Comment by aduprat** [comment]
```
[ERROR] Errors: 
[ERROR]   SpamAssassinContract.expungingSpamMessageShouldNotImpactSpamAssassinState:477 » ConditionTimeout
```
test this please

**Comment by aduprat** [comment]
```
[ERROR] Errors: 
[ERROR]   SpamAssassinContract.imapMovesToSpamMailboxShouldBeConsideredAsSpam:230 » ConditionTimeout
```
test this please

File: mpt/impl/imap-mailbox/external-james/pom.xml [hunk 1]
```diff
@@ -68,4 +68,35 @@
             <artifactId>commons-lang3</artifactId>
         </dependency>
     </dependencies>
+
+    <profiles>
+        <profile>
+            <id>dev</id>
```
Comment (rouazana) [inline mpt/impl/imap-mailbox/external-james/pom.xml]: this profile name is strange, no? If you are in dev, you want to be able to play easily all the tests, no?

Comment (rouazana) [reply mpt/impl/imap-mailbox/external-james/pom.xml]: ok, I will merge this then

**Pull request merged**